templates/registry-standalone/manifest.yaml

```diff
@@ -63,6 +63,14 @@ variables:
   windows_registry:
     type: string
     description: "Flag to add windows images to the registry"
+  download_url:
+    type: string
+    description: "The download URL for all rancher registry resources. i.e. rancher-images.txt"
+    optional: true
+  suse_registry:
+    type: string
+    description: "the suse registry name, if any"
+    optional: true
```

Comment:
Could we match this to what is currently in templates/rancher? ie
  ```
rancher_chart_repo:
    type: string
    default: "latest"
    description: "Name of Helm chart to use for Rancher install. Example: latest"
  rancher_chart_url:
    type: string
    default: "https://releases.rancher.com/server-charts"
    description: "URL of Helm chart to use for Rancher install. Example: https://releases.rancher.com/server-charts"
```

Reply:
udpated the rancher_chart_url variable